app/strings.php

```diff
@@ -0,0 +1,1365 @@
+            __('Community Connectors'),
+            __('Individuals or Community Organizations that can help find Consultation Participants for projects.'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A10473
+        'Explore/individuals' => [
+            __('Individuals'),
+            __('People with lived experience of disability or being Deaf.'),
+            __('Search by name...'),
+            __('Apply filters'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1309%3A18647
+        'Explore/individuals/search results' => [
+            __(':count results for'),
+            __('Search term'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1309%3A18938
+        'Explore/individuals/filters' => [
+            __('Filters'),
+            __('Clear filters'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1301%3A17738
+        'Explore/CommOrgs' => [
+            __('Community organizations'),
+            __('Disability and Deaf organizations, and civil societies.'),
+            __('Type of community organization'),
+            __('Representative organization'),
+            __('Support organization'),
+            __('Civil societies'),
+            __('Read more about these types of organizations'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2221%3A61802
+        'Explore/AC' => [
+            __('Search name or lived experience...'),
+            __('Individual or organization'),
+            __('Organizations'),
+            __('Sectors of interest'),
+            __('Transportation'),
+            __('Financial services'),
+            __('Telecommunication'),
+            __('Broadcasting'),
+            __('Federal government programs and services'),
+            __('Crown corporation'),
+            __('Employment'),
+            __('Buildings and public spaces'),
+            __('Information and communication technology'),
+            __('Communication, other than information and communication technologies'),
+            __('Buying goods and services'),
+            __('Providing programs and services'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2733%3A77694
+        'Explore/CC' => [
+            __('Ethnoracial group they can connect to'),
+            __('First Nations'),
+            __('Inuit'),
+            __('Métis'),
+            __('East Asian'),
+            __('South Asian'),
+            __('Southeast Asian'),
+            __('Latin American'),
+            __('African'),
```

Comment:
These are already present in `IndigenousIdentitySeeder` and `EthnoracialIdentitySeeder`